osf/models/preprint_provider.py

```diff
@@ -14,21 +14,19 @@
 
 class PreprintProvider(ObjectIDMixin, BaseModel):
```

Comment:
If you did the array field you could validate each string individually, you'd just have to concatenate the results when you need the full string. The more I think about the comma-delimited string the more I feel bad about myself for suggesting it.